src/QueryStrategies/BatchQueryStrategies/AllRandomBatchQs.jl

```diff
@@ -0,0 +1,20 @@
+struct AllRandomBatchQs <: BatchPQs
+    k::Int
+
+    function AllRandomBatchQs(;k::Int=0)::AllRandomBatchQs
```

Comment:
This signature and the check for `k<1` is basically a requirement to set k. So this should either be
```julia
function AllRandomBatchQs(;k)::AllRandomBatchQs
```
or just 
```julia
function AllRandomBatchQs(k)::AllRandomBatchQs
```
I'd prefer the second option, or is there any reason why we need a keyword argument?

Reply:
The variable query strategy parameters are passed to `initialize_qs` in a dictionary which then calls the specific Qs constructor. We can do Option 1.